src/main/java/javaserver/ResponseBuilders/OptionResponseBuilder.java

```diff
@@ -0,0 +1,51 @@
+package javaserver.ResponseBuilders;
+
+import java.util.Arrays;
+
+import javaserver.HTTPStatusCode;
+import javaserver.Request;
+import javaserver.Response;
+import javaserver.Routes;
+
+public class OptionResponseBuilder implements ResponseBuilder {
+
+	private Response response;
+
+	@Override
+	public Response getResponse(Request request) {
+		this.response = new Response();
```

Comment:
I think the suggestion of the Builder pattern for the Response will be pretty useful for these objects. You could eliminate the reference to the Response object, and your getResponse could be rewritten as below:
```java
public Response getResponse(Request request) {
  return new Response.Builder()
    .setStatusLine(getStatusLine(request))
    .setHeader(getResponseHeader(request))
    .build();
}
```